locales/en.default.schema.json

```
@@ -1568,6 +1568,36 @@
       "presets": {
         "name": "Rich text"
       }
+    },
+    "video": {
+      "name": "Video",
+      "settings": {
+        "heading": {
+          "label": "Heading"
+        },
+        "cover_image": {
+          "label": "Cover image"
+        },
+        "video_url": {
+          "label": "URL",
+          "placeholder": "Use a YouTube or Vimeo URL",
+          "info": "Video will play inside the page."
+        },
+        "description": {
+          "label": "Video alt text",
+          "info": "Describe the video for customers using screen readers"
```

```suggestion
          "info": "Describe the video for customers using screen readers."
```
Just add a period to the end of info text please!